core/src/main/java/org/mskcc/cbio/portal/util/ExtendedMutationUtil.java

```diff
@@ -175,8 +179,8 @@ public static boolean isValidProteinChange(String proteinChange)
 		boolean invalid = proteinChange == null ||
 		                  proteinChange.length() == 0 ||
 		                  proteinChange.equalsIgnoreCase("NULL") ||
-		                  proteinChange.equalsIgnoreCase(NOT_AVAILABLE);
-
+		                  proteinChange.equalsIgnoreCase(NOT_AVAILABLE.get(0)) ||
```

Comment:
This can be changed to `proteinChange.upper() in map(str.upper, NOT_AVAILABLE)` or something like that instead of having a discrete check for each item in the list.